dotCMS/src/integration-test/java/com/dotmarketing/portlets/languagesmanager/business/LanguageFactoryIntegrationTest.java

```diff
@@ -53,6 +54,7 @@ public void test_get_languages(){
          assertTrue(languages.size() >= 2);
     }
 
+    @Ignore("English isn't necessarily the default language. ")
```

Comment:
if this test doesn't apply anymore, I'd remove it instead of ignoring it

Reply:
Now that I restored the language in a finally the default lang should be US english.